src/components/GameView/ChooseTwoDialog.vue

```diff
@@ -0,0 +1,90 @@
+<template>
```

Comment:
I split the `CounterDialog` into this component (`ChooseTwoDialog`) and `ChooseWhetherToCounterDialog` to reduce the amount of mixing and matching. There was overlap between the two but it was messy and weird to keep straight so this way there's just a separate dialog for asking whether you'd like to counter (the other one) and this one for picking which two to use once you've decided to counter

Reply:
This is a tough one but naming is always hard. Web components are recommended to have a 2 word name. PascalCase is also the recommended format for referencing them in the template so we may want a follow up ticket to go through and rename everything. 😬
- https://vuejs.org/guide/components/registration.html#local-registration

This is Vue 2, but still interesting and informative:
- https://v2.vuejs.org/v2/style-guide/?redirect=true#Multi-word-component-names-essential

Reply:
I thought the recommendation was just that the name be multi-word not exactly 2. Don't these names conform to that rule?

I've created https://github.com/cuttle-cards/cuttle/issues/396 to swap to PascalCase template instances. Better to tackle that one in one fell swoop in a separate PR